apps/studio/pages/project/[ref]/logs/explorer/index.tsx

```diff
@@ -130,9 +179,49 @@ export const LogsExplorerPage: NextPageWithLayout = () => {
 
   const handleRun = (value?: string | React.MouseEvent<HTMLButtonElement>) => {
     const query = typeof value === 'string' ? value || editorValue : editorValue
+
     if (value && typeof value === 'string') {
       setEditorValue(value)
     }
+
+    if (sourceType === 'warehouse') {
+      const whQuery = warehouseEditorValue
+
+      if (!warehouseCollections?.length) {
+        toast.error("You don't have any collections in your warehouse yet.")
```

Comment:
lets avoid contractions for consistency
```suggestion
        toast.error("You do not have any collections in your warehouse yet.")
```